multistream-select/src/protocol/error.rs

```diff
@@ -23,6 +23,7 @@
 use std::error;
 use std::fmt;
 use std::io::Error as IoError;
+use varint::Error as VarIntError;
```

Comment:
`VarintError`, or even better just use `io::Error` and `varint::Error`. Renaming them is unnecessarily confusing. I know we've had this conversation before but I think this is a good example as to why I prefer the latter style. Apart from anything else, if we always use this style and then we capitalise in different ways (or choose `StdIoError` because we also imported `tokio::io::Error`) then it becomes difficult to find all usages of this type.